src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,10 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        return (userService.findByEmail(email) == null) ? false :
+                (userService.findByEmail(email).getPassword() == password) ? true : false;
```

Comment:
Here is another possible approach which I prefer more :)
```suggestion
        return userService.findByEmail(email) != null
                && userService.findByEmail(email).getPassword().equals(password);
```